README.md

```diff
@@ -22,8 +22,24 @@ or
 yarn add form-container
 ```
 
+## Getting started
+
+<a id="connectForm"></a>
+### `connectForm([validators], [formConfig])(WrappedComponent)`
+`form-container` exposes `connectForm` function to connect an arbitrary form component (`WrappedComponent`).
+It does not modify the component class passed to it; instead, it *returns* a new, connected component class for you to use.
+
+<a id="connect-form-arguments"></a>
+#### Arguments
+
+* [`validators: ValidationRule[]`] \(*Array*): An array of rules can be provided to validate the form model against them. Rules are executed in a sequence that is defined in the array.  
+
+* [`formConfig: IFormConfig`] \(*Object*): An object contains initial configuration for the form  
```

Comment:
Is it worth mentioning this is optional?

Reply:
I assumed `[]` say that

Reply:
Ah. Wasn't aware of that convention, though I have a really difficult time reading method signatures like this.

lgtm